hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/ListInfoSubcommand.java

```diff
@@ -51,6 +51,16 @@
       defaultValue = "")
   private String uuid;
 
+  @CommandLine.Option(names = {"--NodeOperationalState"},
+      description = "Show info by datanode NodeOperationalState.",
```

Comment:
Please add all available NodeOperationalState values in the description to asssit user's better choice.